woocommerce-myparcel.php

```diff
@@ -214,6 +215,13 @@ public function initialize(): void
                 'wp_ajax_' . WCMYPA_Settings::SETTING_TRIGGER_MANUAL_UPDATE,
                 [AccountSettings::class, "restRefreshFromApi"]
             );
+
+            // Show temporary message concerning insurances for shipments to Belgium.
```

Comment:
misschien een TODO erbij met wanneer hij weggehaald moet worden, dan is ie wat makkelijker te vinden